### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_RestartKernelPopUp_WAT: Color Contrast is less than 4.5:1 for Close button

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4001]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106
Operating System:Windows10

**Repro Steps:**
1.Login with the appropriate credentials to azure notebook account.
2. Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3. Click on Run or right click and select Run to run the notebook.
4. Navigate   to run button & hit enter -->Select any file & hit enter -->Navigate to Kernel menu --> select Restart & Run all.
5. Open WAT Tool.
6. Open Color Contrast Analyser.
7. Verify Color Contrast for Close Button.

**Actual:**
Color Contrast is less than 4.5:1 for Close Button.

**Expected:**
Color Contrast should be >= 4.5:1 for Close Button.

**User Impact:**
If ColorContrast is <4.5:1 than Low Vision Users face difficulty to See the Close Button.

**Attachment:**
![a11y_azurenotebooks_jupyter_RestartKernelPopUp_wat](https://user-images.githubusercontent.com/39204909/45815040-2c3ced80-bcf5-11e8-8bfa-27ba3da3ec40.png)

**Comment by takluyver** [comment]
Fixed by #4679
